pkg/services/login/login.go

```diff
@@ -197,9 +202,10 @@ func syncOrgRoles(user *models.User, extUser *models.ExternalUserInfo) error {
 	for _, org := range orgsQuery.Result {
 		handledOrgIds[org.OrgId] = true
 
-		if extUser.OrgRoles[org.OrgId] == "" {
+		extRole := extUser.OrgRoles[org.OrgId]
+		if extRole == "" {
 			deleteOrgIds = append(deleteOrgIds, org.OrgId)
-		} else if extUser.OrgRoles[org.OrgId] != org.Role {
+		} else if extRole != org.Role {
```

Comment:
minor remark: you can use the same variable `extRole` for constructing the `cmd` [below](https://github.com/grafana/grafana/pull/26716/files#diff-6db13569d0b1313cbb06a85d465729eaR210)

Reply:
Thanks @papagian - having a look.

Reply:
Done.